moveit_ros/perception/mesh_filter/src/gl_renderer.cpp

```diff
@@ -246,12 +246,12 @@ const GLuint& mesh_filter::GLRenderer::getProgramID() const
   return program_;
 }
 
-const double& mesh_filter::GLRenderer::getNearClippingDistance() const
+double mesh_filter::GLRenderer::getNearClippingDistance() const
 {
   return near_;
 }
 
-const double& mesh_filter::GLRenderer::getFarClippingDistance() const
```

Comment:
This also has the potential to change semantics and can potentially be an API break if such a change happens in a public function. It's valid to store this return value as a `const&` but that becomes invalid after this transformation. This is not a big problem per se but such things ought to be considered.